### ISSUE TITLE
buzzer.cfg

### ISSUE THREAD

**Issue by gilbrotheraway** [dudeofawesome/klipper-config#3]
half of the songs don´t stop at the end

had to add this for the macro to work
[output_pin beeper]
pin: PC0
pwm: true

**Comment by dudeofawesome** [comment]
Yeah, there are definitely some issues around the songs unfortunately. Klipper doesn't allow for changing settings on PWM pins frequently enough to allow for the songs to play at full speed either (I think there's a PR around this)

Is this the configuration you're thinking of?
https://github.com/dudeofawesome/klipper-config/blob/35f9022246322c5e31bff8b99a4ec3ef67c6c153/skr-mini-e3-v1.2.cfg#L171-L176
